src/typeDefs/inputs.ts

```diff
@@ -144,7 +144,7 @@ export const inputs = gql`
   input OrganizationInput {
     name: String!
     description: String!
-    location: String
+    address: AddressInput
```

Comment:
@palisadoes can organizations choose to give no address information?

Reply:
Make it mandatory

Reply:
@palisadoes ok sir. I will change it to mandatory.

Reply:
@palisadoes @xoldyckk Sir, I have  made the address field mandatory for creation of organizations. Thank You.

Reply:
@palisadoes which of the address fields [here](https://github.com/PalisadoesFoundation/talawa-api/blob/46771dbdb3013f9a548a5b331cd2247e26032462/src/typeDefs/types.ts#L361) should be mandatory?